tests/fast-integration/skr-test/skr-test.js

```diff
@@ -68,6 +80,21 @@ describe("SKR test", function() {
   it("order.created.v1 event should trigger the lastorder function", async function () {
     await sendEventAndCheckResponse();
   });
+
+  it("Deletes the resources that have been created", async function () {
+    await deleteMockTestFixture("mocks", testNS);
+  });
+
+  // Check audit log for AWS
+  if (process.env.KEB_PLAN_ID == AWS_PLAN_ID) {
+    it ("Check audit logs", async function() {
+      await checkAuditLogs(auditlogs)
+    })
+
+    it ("Amount of audit events should not exceed a certain threshold", async function() {
```

Comment:
"should" as in "recommendation", or is it rather a "must" as a hard limit?

"Should" is rather ambiguous and it may often be better to it phrase it differently.